### DISCUSSION TITLE
User reload issue

### DISCUSSION THREAD

**Discussion by sumasagrp** [vuejs/vuefire#1465] [Questions and Help]
user has the method/property to load reload user . However calling this method 

The idea was to reload the user to update the emailverified property because it doesn't seem to update. 

Is this a vuefire issue ?

```
<script setup>
import { useCurrentUser } from 'vuefire'

const user = useCurrentUser()

user.value.reload()
</script>
```

**Comment by posva** [comment]
It seems like your sentence is infinished. What's the issue with the `reload()` method?
